code/script.js

```diff
@@ -34,18 +59,164 @@ const showMessage = (message, sender) => {
 }
 
 // Starts here
+// Greeting 
 const greetUser = () => {
   // here we call the function showMessage, that we declared earlier with the argument "Hello there, What's your name?" for message, and the argument "bot" for sender
-  showMessage("Hello there, What's your name?", 'bot');
+  showMessage("Hi! Welcome to our amazing plant store! What's your name?", 'bot');
   // Just to check it out, change 'bot' to 'user' here 👆
 }
+//User answering their name
+const handleInput = (event) => {
+  event.preventDefault(); // prevents refreshing the page
+  let userName = nameInput.value;
+  showMessage(`Hi! I'm ${userName}.`, "user");  //calling the function
+  nameInput.value = ""; // erases the answer field
+ 
+  setTimeout(() => question1(userName), 500);
+  
+};
 
-// Set up your eventlisteners here
+//Bot - question 1
+const question1 = (userName) => {
+  showMessage(`Aloha ${userName}! How have you been? Good, fantastic, surviving, huh?`, "bot");
+  nameForm.removeEventListener("submit", handleInput);
+  nameForm.addEventListener("submit", handleInput2);
+  
+};
+
+// User - answer1
+const handleInput2 = (event) => {
+  event.preventDefault();
+  let moodType = nameInput.value;
+  showMessage(`${moodType}`, "user"); //calling the function
+  nameInput.value = "";
+  setTimeout(() => question2(), 500);
+
+}
+
+//Bot - question 2
+const question2 = (event) => {
+  showMessage(`Music to my ears! I assume you need more plants?`, "bot");
+  inputWrapper.innerHTML = `
+  <button id="Yes" type="submit" class="chat-btn"> Yes, of course 🤩</button>
+  <button id="No" type="submit" class="chat-btn"> It's so sorrowful, but actually no 😔 </button> `
+
+  document.getElementById("Yes").addEventListener("click",yesChoice);
+  document.getElementById("No").addEventListener("click",noChoice);
+};
+
+//User - answer 2 - no
+const noChoice = (event) => {
```

Comment:
It took me a while to understand how you connected the click on the no-button with this next part, but I see the noChoice now in both parts. Smartly done!